[CI] Failures in DocsClientYamlTestSuiteIT {yaml=reference/snapshot-restore/apis/get-snapshot-status-api/line_309}

<!--
Please fill out the following information, and ensure you have attempted
to reproduce locally
-->

**Build scan**:

https://gradle-enterprise.elastic.co/s/gs2c3e636a73s
https://gradle-enterprise.elastic.co/s/di5pmdh26jj2y

**Repro line**:

    ./gradlew ':docs:integTestRunner' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/snapshot-restore/apis/get-snapshot-status-api/line_309}" -Dtests.seed=580EE8BCEA3F32A5 -Dtests.security.manager=true -Dtests.locale=ar-BH -Dtests.timezone=Etc/GMT+1 -Druntime.java=11

**Reproduces locally?**:

Yes!

**Applicable branches**:

`master`

**Failure history**:

[Gradle enterprise history](https://gradle-enterprise.elastic.co/scans/tests?search.buildToolType=gradle&search.buildToolType=maven&search.relativeStartTime=P7D&search.timeZoneId=Europe/London&tests.container=org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT&tests.sortField=FAILED&tests.test=test%20%7Byaml%3Dreference/snapshot-restore/apis/get-snapshot-status-api/line_309%7D&tests.unstableOnly=true)

**Failure excerpt**:

<details>
<summary>Stack trace / error</summary>

```
        java.lang.AssertionError: $body didn't match expected value:
                                 $body: 
                               snapshots: 
                                         0: 
                        include_global_state: same [false]
                                     indices: 
                                       index_1: 
                                          shards: 
                                                 0: 
                                               stage: same [DONE]
                                               stats: 
                                           incremental: 
                                              file_count: same [0]
                                           size_in_bytes: same [0]
                                  start_time_in_millis: same [1594989648341]
                                        time_in_millis: same [0]
                                                 total: 
                                              file_count: expected Integer [4] but was Integer [7]
                                           size_in_bytes: same [11590]
                                    shards_stats: 
                                              done: same [1]
                                            failed: same [0]
                                        finalizing: same [0]
                                      initializing: same [0]
                                           started: same [0]
                                             total: same [1]
                                           stats: 
                                       incremental: 
                                          file_count: same [0]
                                       size_in_bytes: same [0]
                              start_time_in_millis: same [1594989648341]
                                    time_in_millis: same [0]
                                             total: 
                                          file_count: expected Integer [4] but was Integer [7]
                                       size_in_bytes: same [11590]
                                  repository: same [my_repository]
                                shards_stats: 
                                          done: same [1]
                                        failed: same [0]
                                    finalizing: same [0]
                                  initializing: same [0]
                                       started: same [0]
                                         total: same [1]
                                    snapshot: same [snapshot_2]
                                       state: same [SUCCESS]
                                       stats: 
                                   incremental: 
                                      file_count: same [0]
                                   size_in_bytes: same [0]
                          start_time_in_millis: same [1594989648341]
                                time_in_millis: same [201]
                                         total: 
                                      file_count: expected Integer [4] but was Integer [7]
                                   size_in_bytes: same [11590]
                                        uuid: same [gHeds_gQSrubBFRlf7MUvg]
```

</details>

--- a/docs/reference/snapshot-restore/apis/get-snapshot-status-api.asciidoc
+++ b/docs/reference/snapshot-restore/apis/get-snapshot-status-api.asciidoc
@@ -59,7 +59,7 @@ PUT /_snapshot/my_repository/my_snapshot?wait_for_completion=true
 
 PUT _snapshot/my_repository/snapshot_2?wait_for_completion=true
 {
-  "indices": [],
+  "indices": ["index_1"],
   "ignore_unavailable": true,
   "include_global_state": false,
   "metadata": {
@@ -392,5 +392,6 @@ GET /_snapshot/my_repository/snapshot_2/_status
 // TESTRESPONSE[s/"start_time_in_millis" : 1594829326691/"start_time_in_millis" : $body.snapshots.0.stats.start_time_in_millis/]
 // TESTRESPONSE[s/"time_in_millis" : 205/"time_in_millis" : $body.snapshots.0.stats.time_in_millis/]
 // TESTRESPONSE[s/"file_count" : 3/"file_count" : $body.snapshots.0.stats.incremental.file_count/]
+// TESTRESPONSE[s/"file_count" : 4/"file_count" : $body.snapshots.0.stats.total.file_count/]
 // TESTRESPONSE[s/"size_in_bytes" : 5969/"size_in_bytes" : $body.snapshots.0.stats.incremental.size_in_bytes/]
 // TESTRESPONSE[s/"start_time_in_millis" : 1594829326896/"start_time_in_millis" : $body.snapshots.0.indices.index_1.stats.start_time_in_millis/]

